docs/_js/examples/todo.js

```diff
@@ -1,8 +1,8 @@
 var TODO_COMPONENT = `
 var TodoList = React.createClass({
   render: function() {
-    var createItem = function(itemText) {
-      return <li>{itemText}</li>;
+    var createItem = function(itemText, index) {
+      return <li key={index}>{itemText}</li>;
```

Comment:
Can you actually make this `key={index + itemText}`? While what you have here silences the warning, it doesn't do anything differently so will still hit the same issue as if you excluded keys entirely.